Wikipedia/Code/NotificationsCenterCommonViewModel+ActionExtensions.swift

```diff
@@ -229,7 +240,7 @@ extension NotificationsCenterCommonViewModel {
             return nil
         }
 
-        let data = NotificationsCenterActionData(text: title, url: url, iconType: .document, destinationText: destinationText(for: url))
+        let data = NotificationsCenterActionData(text: title, url: url, iconType: .document, destinationText: destinationText(for: url), actionType: nil)
```

Comment:
I'll confirm to be sure they don't want any action strings for these remaining types - it should be easy to add them since the action takes any `String`. I'll update when I hear back!

Reply:
Thanks for checking that! I was sending `nil` for the actions the schema didn't cover in case it could be somehow useful, but I can see that it might not be the case